client/stylesheets/_modal.scss

```diff
@@ -843,11 +843,15 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	margin-top: 75px !important;
 }
 
-.modFeedbackPageTwo {
+.modFeedbackPageTwo, .modReportPageOne, .modReportPageThree {
```

Comment:
Selector `modFeedbackPageTwo` should be written in lowercase with hyphens<br>Selector `modReportPageOne` should be written in lowercase with hyphens<br>Selector `modReportPageThree` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line